action/container/container.go

```diff
@@ -46,8 +45,27 @@ func Setup(ctx context.Context, client *dagger.Client, opts *SetupOpts) (*dagger
 		return nil, errDirectoryInvalid
 	}
 
-	// Pull docker container
-	container := client.Container().From(opts.ContainerURL)
+	// Setup container either from URL or build from Dockerfile
+	var container *dagger.Container
+	if dockerfileDirectoryPath == "" {
+		// Use URL
+		fmt.Printf("Container setup: URL mode")
```

Comment:
If you want to use the `fmt` package for logging use either `fmt.Printf("MSG\n")` or if you don't need any format specifiers like in this line you can shortcut with `fmt.Println("MSG")`